disnake/ext/commands/common_bot_base.py

```diff
@@ -395,6 +395,14 @@ def cogs(self) -> Mapping[str, Cog]:
         """Mapping[:class:`str`, :class:`Cog`]: A read-only mapping of cog name to cog."""
         return types.MappingProxyType(self.__cogs)
 
+    @property
+    def listeners(self) -> Mapping[str, List[CoroFunc]]:
+        """Mapping[:class:`str`, List[Callable]]: A read-only mapping of event name to listeners.
```

Comment:
```suggestion
        """Mapping[:class:`str`, List[Callable]]: A read-only mapping of event names to listeners.
```

It's not *entirely* read-only since the list values are still mutable, but there's no way around that.
Would be great to mention `.add_listener` and `.remove_listener` for that purpose, though